sonaranalyzer-dotnet/src/Tests/SonarAnalyzer.UnitTest/TestCases/DoNotDecreaseMemberVisibility.cs

```diff
@@ -31,6 +31,12 @@ class A
 
         public void Method_13<T, V>(T obj, IEnumerable<V> obj2) { }
 
+        public void Method_14(int count) { }
+
+        public void Method_15(out int count) { }
+
+        public void Method_16(ref int count) { }
+
```

Comment:
Not sure to have seen a case for method with a `virtual` in base class and a `override` in sub-class with a compliant message. If not there, could you add the test?